src/sx/src/transformValue.js

```diff
@@ -12,6 +12,11 @@ export default function transformValue(
     // https://engineering.fb.com/web/facebook-com-accessibility/
     const defaultFontSize = 16;
     return `${Number(styleValue) / defaultFontSize}rem`;
+  } else if (styleName === 'zIndex') {
+    if (styleValue === 'auto') {
+      return `${styleValue}`;
+    }
+    return `${Number(styleValue)}`;
```

Comment:
This should also be removed. It's going to work just fine once we fix the generator (unless I am missing something). Of course, the generated Flow types will be a bit more imprecise but that's correct. Luckily, we have a way how to make them more precise again - you can overwrite them here: https://github.com/adeira/universe/blob/2f2fa898f442feda1ec0be225ca2e962c230263f/src/sx/src/css-properties/CustomPropertyTypes.js